pom.xml

```diff
@@ -76,17 +82,21 @@
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
-                <version>3.3</version>
+                <version>3.7.0</version>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>1.7</source>
-                    <target>1.7</target>
+                    <source>1.8</source>
+                    <target>1.8</target>
```

Comment:
This is a breaking change for anyone downstream and feels like it should be part of a major version bump, i.e 0.x.x to 1.0.x

Reply:
👍

Reply:
Reverted com.github.mmazi.rescu back to v1.8.1 to allow Java 7 support for this release